.github/workflows/build.yml

```diff
@@ -33,3 +33,13 @@ jobs:
       run: |
         GO111MODULE=off go get github.com/mattn/goveralls
         $(go env GOPATH)/bin/goveralls -coverprofile=profile.cov -service=github
+    - name: Vet
+      run: go vet ./...
+    - name: Install staticcheck
+      run: "GO111MODULE=off go get honnef.co/go/tools/cmd/staticcheck"
```

Comment:
should this be pinned to a specific version?

Reply:
I'm not sure if Go 1.15 supports pinning the version in `go get` -- . Do we actually still need to test against Go 1.15? The `go.mod` specifies 1.16 as the minimum version.

Reply:
Yeah I don't see why we should...

Reply:
Oh hm, I must have started this change on a stale branch then rebased, since 1.15 was just removed a few days ago: https://github.com/theupdateframework/go-tuf/pull/179

I'll remove this